### ISSUE TITLE
[Errno 2] No such file or directory: '~/.local/lib/python3.6/site-packages/sigProfilerPlotting/templates/DBS78.pkl'

### ISSUE THREAD

**Issue by RTCarmo** [AlexandrovLab/SigProfilerExtractor#162]
Hello!

First of all, thanks for the amazing tool!

I'm trying to run the example given [here](https://osf.io/t6j7u/wiki/2.%20Quick%20Start%20Example/).

Extracting steps worked just fine but, at decomposing plot step I got the following error:

> There may be an issue with the formatting of your matrix file.
> Traceback (most recent call last):
>   File "~/.local/lib/python3.6/site-packages/sigProfilerPlotting/sigProfilerPlotting.py", line 4743, in plotDBS
>     fig_orig=pickle.load(open(spplt.__path__[0]+'/templates/DBS78.pkl','rb'))
> FileNotFoundError: [Errno 2] No such file or directory: '/home/rtorreglosa/.local/lib/python3.6/site-packages/sigProfilerPlotting/templates/DBS78.pkl'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/home/rtorreglosa/.local/lib/python3.6/site-packages/SigProfilerExtractor/sigpro.py", line 898, in sigProfilerExtractor
>     sub.export_information(loopResults, m, layer_directory, index, colnames, wall=wall, sequence=sequence)
>   File "~/.local/lib/python3.6/site-packages/SigProfilerExtractor/subroutines.py", line 1726, in export_information
>     plot.plotDBS(signature_subdirectory+"/"+mutation_type+"_S"+str(i)+"_Signatures"+".txt", signature_subdirectory+"/Signature_plot" , "S"+str(i), "78", True, custom_text_upper=stability_list, custom_text_middle=total_mutation_list)
>   File "~/.local/lib/python3.6/site-packages/sigProfilerPlotting/sigProfilerPlotting.py", line 4883, in plotDBS
>     os.remove(output_path + 'DBS_78_plots_' + project + '.pdf')
> FileNotFoundError: [Errno 2] No such file or directory: 'results/DBS78/All_Solutions/DBS78_1_Signatures/Signatures/Signature_plotDBS_78_plots_S1.pdf'

I'm running the following commands:
```
>>> from SigProfilerExtractor import sigpro as sig
>>> sig.sigProfilerExtractor("vcf", "results", "Desktop/test/21BRCA/21BRCA_vcf", reference_genome="GRCh37", minimum_signatures=1, maximum_signatures=10, nmf_replicates=100)
```

Hope you can help me if this issue!
Thanks!

**Comment by rvangara** [comment]
Hi @RTCarmo ,

Please install the latest version of SigProfilerPlotting(v1.3.1) and try it, that should resolve this issue.

Please reopen the ticket if the issue persists.

Regards,
Ravi.

**Comment by RTCarmo** [comment]
Hi @rvangara ,

Thanks for the quick response!

I just checked the version installed, and it's already the latest version.
I just tried to run again, and I got the same problem! Although, running for the matrix file, using:

```
>>> sig.sigProfilerExtractor("matrix", "results", "Desktop/test/21BRCA/21BRCA.txt", reference_genome="GRCh37", minimum_signatures=1, maximum_signatures=10, nmf_replicates=100, cpu=-1)
```
Got me no issues! Maybe there is a problem in the matrix file generated from the vcfs? Do you have any hints?

**Comment by rvangara** [comment]
Hi @RTCarmo 

The error that you posted earlier was coming from a recent update on SigProfilerPlotting. This is fixed in SigProfilerPlotting v1.3.1. Check if that is the version of SigProfilerPlotting that you currently have in your environment.

Ravi

**Comment by RTCarmo** [comment]
Hi @rvangara ,

I just managed to run it on another computer. I don't know the reason why, but it worked properly.

Many thanks for the help!

**Comment by AldhairMedico** [comment]
Hi @rvangara, 
I'm having the same issue:
FileNotFoundError: [Errno 2] No such file or directory: 

> '/home/aldhair/.local/lib/python3.10/site-packages/sigProfilerPlotting/templates/DBS78.pkl'

According to pip3 list:

> SigProfilerAssignment 0.0.18
> SigProfilerExtractor 1.1.4
> SigProfilerMatrixGenerator 1.2.13
> SigProfilerPlotting 1.3.2

My code was: 
from SigProfilerExtractor import sigpro as sig
sig.sigProfilerExtractor("vcf", "59only", "/home/aldhair/Apps/SigProfilerExtractor-1.1.4/59only", reference_genome="GRCh38", opportunity_genome = "GRCh38", exome = True, minimum_signatures=1, maximum_signatures=25, nmf_replicates=100, resample = True, batch_size=1, cpu=-1, gpu=False, nmf_init="nndsvd_min", precision= "single", matrix_normalization= "gmm", min_nmf_iterations= 10000, max_nmf_iterations=1000000, nmf_test_conv= 10000, nmf_tolerance= 1e-15, nnls_add_penalty=0.05, nnls_remove_penalty=0.01, initial_remove_penalty=0.05, de_novo_fit_penalty=0.02, get_all_signature_matrices= False)

I'm working on the Ubuntu subsystem for Windows 11, any idea of what is happening?

**Comment by rvangara** [comment]
Hi @AldhairMedico ,

The error is due to the recent update in the Sigprofilerplotting library. 

However, this issue should be addressed in SigProfilerPlotting 1.3.2 as you are showing in your pip3 list. 

Can you please make sure to upgrade SigProfilerPlotting to the recent version of 1.3.2 and remove any confounding previous versions of this library installed in your environment?

Best,
Ravi
